src/main/java/org/eclipse/uprotocol/mqtt/HiveMqMQTT5Client.java

```diff
@@ -45,19 +48,19 @@
 
 class HiveMqMQTT5Client implements UTransport {
 
-    private static final Logger LOG = LoggerFactory.getLogger(HiveMqMQTT5Client.class);
-    private static final String USER_PROPERTIES_KEY_FOR_ID = "1";
-    private static final String USER_PROPERTIES_KEY_FOR_MESSAGE_TYPE = "2";
-    private static final String USER_PROPERTIES_KEY_FOR_SOURCE_NAME = "3";
-    private static final String USER_PROPERTIES_KEY_FOR_SINK_NAME = "4";
-    private static final String USER_PROPERTIES_KEY_FOR_PRIORITY = "5";
-    private static final String USER_PROPERTIES_KEY_FOR_TTL = "6";
-    private static final String USER_PROPERTIES_KEY_FOR_PERMISSION_LEVEL = "7";
-    private static final String USER_PROPERTIES_KEY_FOR_COMMSTATUS = "8";
-    private static final String USER_PROPERTIES_KEY_FOR_REQID = "9";
-    private static final String USER_PROPERTIES_KEY_FOR_TOKEN = "10";
-    private static final String USER_PROPERTIES_KEY_FOR_TRACEPARENT = "11";
-    private static final String USER_PROPERTIES_KEY_FOR_PAYLOAD_FORMAT = "12";
+    public static final Logger LOG = LoggerFactory.getLogger(HiveMqMQTT5Client.class);
```

Comment:
You should fetch all from the protobuf generated code and not statically define it here.

Reply:
Why should the mqtt specific userproperties keys be defined in the transport independent protobuf files?

Reply:
the source of truth for the values of the keys are in protobuf so you should not hardcode it but fetch from protobuf generated code.

Reply:
Todo:

- [ ] use identifier for uAttributes from Photobuf